controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java

```diff
@@ -462,7 +463,8 @@ public static PersistentResourceXMLBuilder builder(final PathElement pathElement
         protected boolean noAddOperation;
         protected AdditionalOperationsGenerator additionalOperationsGenerator;
         protected final LinkedList<AttributeDefinition> attributeList = new LinkedList<>();
-        protected final List<PersistentResourceXMLBuilder> children = new ArrayList<>();
+        final List<PersistentResourceXMLBuilder> childrenBuilder = new ArrayList<>();
+        final List<PersistentResourceXMLDescription> children = new ArrayList<>();
```

Comment:
This changes the visibility of 'children'.

It's too bad this class isn't final and all of these aren't just private. There are no subclasses in core or full.